spec/graphql/types/user_type_spec.rb

```diff
@@ -37,8 +34,8 @@
     create(:comment, author: user, body: "Public")
     create(:budget_investment_comment, author: user, valuation: true, body: "Valuation")
 
-    response = execute("{ user(id: #{user.id}) { public_comments { edges { node { body } } } } }")
-    received_comments = dig(response, "data.user.public_comments.edges")
+    response = run_graphql_field("User.public_comments", user)
+    received_comments = response.items.map { |comment| { "node" => { "body" => comment.body }} }
```

Comment:
Same here: this test would pass even if `Types::CommentType` didn't have a `body` field.